survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) is exploring the possibility of allowing the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) for the HTML content document format in EPUB 3.4, instead of restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it is in earlier versions of EPUB 3.
+
+This initiative aims to address the diminishing use and maintenance of the XML syntax outside the publishing industry while leveraging the future evolution of HTML. 
+
+The proposal does not change the document formats currently allowed in an EPUB. Since EPUB is already reliant on HTML, this proposal merely removes the restriction to the XML syntax.
+
+PMWG is seeking your valuable feedback. What can you see as potential benefits? Do you anticipate any technical challenges? What would you consider a reasonable implementation timeline?  Your insights will help shape the future of EPUB standards. 
+
+Your responses will be kept anonymous but will be made public to ensure transparency. Providing some context about your tools and workflows will help us better understand and address your feedback.
+
+**Question 1:** What is your position in the industry and how do your workflows involve EPUB?
+
+**Question 2:** What potential benefits and challenges do you anticipate when incorporating HTML syntax into your processes? 
+
+**Question 3:** Are there any technical, operational, or resource related issues that need to be considered?
```

Comment:
```suggestion
**Question 3:** Are there any technical, operational, or resource related steps that the Working Group should consider to ease the transition (e.g., additional metadata, profiling the EPUB specification, introducing several EPUB versions, etc.)?
```

Reply:
Should we be that verbose and directive here? Simple questions are easier to understand.

Reply:
Yes, that is correct, but I am a little bit afraid the answers will be all over the place. What about

> Are there steps that the WG should consider easing the transition (e.g., additional metadata, profiles of EPUB, etc.)?